dist/main.css

```diff
@@ -0,0 +1,87 @@
+body {
+    background: url(assets/back.jpg) no-repeat center center fixed;
+    background-size: cover;
+}
+
+.div-content {
+    display: flex;
+    justify-content: center;
+}
+
+.card {
+    width: 18rem;
+}
+
+.nav,
+.main {
+    margin: 0 auto;
+    background-color: rgb(217, 227, 236);
+    border: 1px solid gray;
+}
+
+#add-task-btn {
```

Comment:
- [ ] Don't use ids as selectors. This can override other properties and make code unclean.